xe-daemon/xe-daemon.go

```diff
@@ -23,6 +23,10 @@ const (
 	DivisorOne           int    = 1
 	DivisorTwo           int    = 2
 	DivisorLeastMultiple int    = 2 // The least common multiple, ensure every collector done before executing InvalidCacheFlush.
+	SuspendStatusPath    string = "control/process_suspend_status"
+	SysFreezeTimeoutPath string = "/sys/power/pm_freeze_timeout"
+	DefaultTimeout       string = "20000"
```

Comment:
Is it possible customer customized the default value for their VM?

Reply:
Aha, good question, so we may need to check if the pm_freeze_timeout is default value of not first, if it is not 20000 as https://github.com/torvalds/linux/blob/55027e689933ba2e64f3d245fb1ff185b3e7fc81/kernel/power/process.c#L26 kernel set, then we may need to skip to watch this entry